cnd/src/http_api/problem.rs

```diff
@@ -54,11 +53,6 @@ pub fn from_anyhow(e: anyhow::Error) -> HttpApiProblem {
         return HttpApiProblem::new("Swap not found.").set_status(StatusCode::NOT_FOUND);
     }
 
-    if e.downcast_ref::<SwapExists>().is_some() {
```

Comment:
Are we not sending an error specific to existing swap anymore?
Can we please re-introduce the functionality?

Reply:
The error struct was never used outside of this line so I deleted it. Meaning there is no change in functionality with this PR :)

I am not sure this PR is the best place to introduce this functionality again.

I am gonna re-open https://github.com/comit-network/comit-rs/issues/2490.